api/requirements.in

```diff
@@ -38,4 +38,6 @@ analytics-python
 django-lifecycle
 drf-writable-nested
 django-filter
-dataclasses
\ No newline at end of file
+dataclasses
+git+git://github.com/Flagsmith/flagsmith-engine.git@main
+boto3
```

Comment:
Do we need to bring all of boto3 into the requirements? There's no way of specifying just a DynamoDB client? Just aware of how large that package is...

Reply:
We already bring this in for SES so not an issue